docs/0.4/concepts/security_considerations.md

```diff
@@ -0,0 +1,164 @@
+# Security Considerations
+
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+Splinter provides several levels of security and customizable permissions.
+
+## Cryptographic signing with public/private keys
+
+Like Sawtooth and other distributed ledgers, Splinter uses cryptographic
+signing with public/private keys. Each user and transaction-submitting entity
+has a public key and an encrypted private key.
+
+All transactions that affect shared state are signed with the encrypted private
+key. This includes circuit creation (proposals and votes) and smart contract
+operations (uploads, configuration, and changes), as well as functions
+performed by the Splinter daemon and application-specific services.
+
+## User and credential management with Biome
+
+Biome is the Splinter component that manages the relationship between
+user-based data and Splinter's key-based user information. Put simply, Biome
+maps user IDs and passwords (or other authentication credentials) to the
+public/private keys that the rest of Splinter uses to sign transactions.
+
+Biome isolates all user-based data from the rest of Splinter, using a
+Biome-specific database to store the mapping between user data and keys. All
+other Splinter functions use only key-based signing to identify actors and
+verify that they have permission for the requested operation.
+
+It's important to note that Splinter does not encrypt private keys. The client
+application is responsible for encryption before submitting a private key. For
+an example of client-encrypted keys, see the login functionality in the
+Gameroom application (as described in [Gameroom Walkthrough: Behind scene
+1](../examples/gameroom/walkthrough/#i-1-behind-scene-1-alice-logs-into-acmes-gameroom-ui).
+
+**NOTE**: Currently, Biome doesn't support integration with an organization's
+identity provider (such as Active Directory). As a result, the current version
+of Splinter is not appropriate for production deployment.
```

Comment:
Replace Splinter with Biome, to scope the statement about suitability.

Reply:
Fixed.